backend/src/db/mongoose.js

```diff
@@ -1,9 +1,12 @@
 import mongoose from 'mongoose';
+import { envVariables } from '../config';
```

Comment:
maybe you can just get the DATABASE_URL itself instead of whole object

Reply:
Yes, I realized it, now I have changed it to DATABASE_URL. 😀